src/Layout/Footer.tsx

```diff
@@ -252,7 +252,7 @@ function LanguageModal({
                   loadPaymentSetup({
                     projectGUID: router.query.to,
                     paymentSetupCountry: country,
-                    shouldSetPaymentDetails: false,
+                    shouldSetPaymentDetails: true,
```

Comment:
The reason we had set this to `false` is, that we did not want to set paymentDetails unnecessarily (as we only want to change the language here). It would cause many issues afterwards. We have an useEffect in the QueryParamsContext which has paymentDetails in the dependency array. We need to find a different solution for this.

Reply:
But its only paymentOptions Api which returns the translations of name and description of the project
How do u suggest we should handle this?

Reply:
I did this because even after changing language
two important property is currency and effectiveCountry remains the same

what I have noticed is only `name` and `description` properties changes other than that rest is same